renew/renew.go

```diff
@@ -21,6 +21,12 @@ var routes []lib.Route = []lib.Route{
 		Handler: lib.ResponseLoggerWrapper(PromoteFx),
 		Roles:   []string{},
 	},
+	{
+		Route:   "/v1/rename-me",
```

Comment:
we could use `renew-notice` for now while we don't come up with a better name.

Reply:
@Yousef-Hammar the thing is that we are already at the `renew` root i.e.: the path currently is `host/renew/v1/renew-notice`. What we could do would be further split the path as so -> `host/renew/v1/notice/e-commerce` since in the other task #246 the goal is to send the notice for the network, and a possible path would be `host/renew/v1/notice/network`. What do you think?